### PR TITLE
Create RELEASE.md

### PR DESCRIPTION
**Pull request by jost-s**
### TODO:
- [ ] CHANGELOGs updated with appropriate info
- [ ] All code changes are reflected in docs, including module-level docs

### CODE REVIEW

**Review by ThetaSinner — APPROVED** [review]
Nice!

Very good to have this visible outside MM and so that we have an easy to find place to refer to

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    - 10+ second average reception time: unacceptable (fail)
+    - Average reception time is estimated loosely based on 10 or more attempts in one or more sessions, measured with a stopwatch. "Update time" is defined as the time it takes for one node to "store" an image, and another node or nodes to register the hash for that stored image.
+    - Smoke testing is not exhaustive, but an indicator that a release candidate and its dependencies compile, and that basic functionality is confirmed working.
+- If smoke test passes, full release of the release candidate version is approved.
+- If stress test fails, fix and update and release a new rc version and perform smoke test again.
+- Once smoke test passes, and full version is released, bump versions in downstream components and re-publish happs, and initiate full testing. This includes the demo apps Talking Stickies and Kando.
+
+## Full release test
+
+- After release of full version, test again as before with the release candidate and also start a test of persistent DHT functionality that lasts approx. 2 weeks.
```
Comment (mattyg) [inline RELEASE.md:21]: I like the idea of this but would want specific criteria for how the persistent DHT is tested through/after the 2 weeks, similar to the smoke test criteria listed above.

File: RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+- Once testing passes and persistent DHT is functional for the given time period and no bugs have been reported, declare release as "stable".
```
Comment (mattyg) [inline RELEASE.md:22]: What does it mean to declare release as stable? Is this marking the github release as "Latest"? Officially recommending it in public communications?

**Review by mattyg — APPROVED** [review]

File: RELEASE.md (hunk 2 above)
Comment (jost-s) [reply RELEASE.md:22]: Yes, I guess that means it becomes the recommended version. I'll change that.

File: RELEASE.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+## Release candidate test
+
+- Implement fixes, features, changes, and create a release candidate version in the format x.x.x-rc.x
+- Compile a version of the ziptest application in the standalone kangaroo pouch single app runtime using this x.x.x-rc.x version
+- Perform smoke test:
+  - Add images under 50 KiB
+  - Entries must be received by other peers
+    - 0-3 second average reception time: timely (pass)
+    - 3-10 second average reception time: delayed (warn)
```
Comment (jost-s) [inline RELEASE.md:11]: What does "warn" do? Who do we warn? I propose to remove this and extend `pass` to 0-10 seconds.

File: RELEASE.md (hunk 1 above)
Comment (jost-s) [reply RELEASE.md:21]: I agree, that needs to be specified.

**Comment by jost-s** [comment]
@ThetaSinner @mattyg Wrote detailed smoke test procedure, please review again.

**Review by mattyg — APPROVED** [review]
Thanks for this - looking forward to making this automated, but glad to have an explicit process!

**Pull request merged**